public/locales/pt/videoCard.json

```diff
@@ -0,0 +1,3 @@
+{
+    "content": "AletheiaFact.org é um movimento e plataforma coletivo de verificação de fatos que imagina uma"
```

Comment:
The text is cut

Reply:
This is the text that we have in figma.

Reply:
Full text:
```
AletheiaFact.org is a movement and collective platform for fact-checking that imagines a society where everyone can have free access and engage with truthful and credible information with autonomy
```
See translations in the about page.